tests/gordo_components/builder/test_builder.py

```diff
@@ -430,10 +433,10 @@ def test_provide_saved_model_caching_handle_existing_same_dir(tmp_dir):
     data_config = get_random_data()
     output_dir = os.path.join(tmp_dir, "model")
     registry_dir = os.path.join(tmp_dir, "registry")
-
-    builder = ModelBuilder(
-        name="model-name", model_config=model_config, data_config=data_config
+    machine = Machine(
+        name="model-name", dataset=data_config, model=model_config, project_name="test"
```

Comment:
Small thing, but sometimes it is:
```python
name="model-name", project_name="test"
```
and sometimes:
```python
name="test", project_name="test"
```
maybe make these uniform.